tools/docker/Dockerfile.build-env-rocm

```diff
@@ -1,43 +1,24 @@
-FROM rocm/dev-ubuntu-18.04:latest
+FROM rocm/dev-ubuntu-20.04:4.0.1
+
+ENV DEBIAN_FRONTEND=noninteractive
 
 # install compilers, libraries & co
-RUN apt-get update
-RUN apt-get install -y \
-        locales \
-        gfortran \
-        gcc-7 \
-        g++-7 \
-        llvm-7-dev \
-        llvm-7-tools \
-        mpich \
-        libomp-7-dev \
+RUN apt-get update -qq \
+    && apt-get install --no-install-recommends -qq \
+        ca-certificates \
+        gfortran g++ gcc \
+        git \
+        hipblas \
         libmpich-dev \
-        libopenblas-dev \
-        wget
-
-# install rocm libraries
-RUN wget -q -O - http://repo.radeon.com/rocm/rocm.gpg.key | sudo apt-key add -
-RUN echo 'deb [arch=amd64] http://repo.radeon.com/rocm/apt/debian/ xenial main' | sudo tee /etc/apt/sources.list.d/rocm.list
-RUN apt-get update
-RUN apt-get install -y \
-        rocm-dev \
+        libopenblas-openmp-dev \
+        mpich \
+        ninja-build \
         rocblas \
         rocsolver \
-        hipblas
-
-# install git 2.18+
-RUN apt-get install -y software-properties-common
-RUN add-apt-repository ppa:git-core/ppa
-RUN apt-get install -y git
-
-# install ninja
-RUN apt-get install -y wget
-RUN wget https://github.com/Kitware/ninja/releases/download/v1.10.0.gfb670.kitware.jobserver-1/ninja-1.10.0.gfb670.kitware.jobserver-1_x86_64-linux-gnu.tar.gz
-RUN tar -xzvf ninja-1.10.0.gfb670.kitware.jobserver-1_x86_64-linux-gnu.tar.gz
-ENV PATH="/ninja-1.10.0.gfb670.kitware.jobserver-1_x86_64-linux-gnu:${PATH}"
-
-# install cmake
-RUN wget https://github.com/Kitware/CMake/releases/download/v3.17.0/cmake-3.17.0-Linux-x86_64.tar.gz
-RUN tar -xzvf cmake-3.17.0-Linux-x86_64.tar.gz
-ENV PATH="/cmake-3.17.0-Linux-x86_64/bin:${PATH}"
+        wget \
+        python3-pip \
+    && rm -rf /var/lib/apt/lists/* \
+    && wget -qO- "https://cmake.org/files/v3.17/cmake-3.17.5-Linux-x86_64.tar.gz" | \
+       tar --strip-components=1 -xz -C /usr/local \
```

Comment:
just discovered this one here: https://apt.kitware.com/ ... what do you think?

Reply:
Hm yeah, I think it can't hurt to install a specific version and I'm too lazy to figure out how you would pin it with apt